Plaid API Endpoints Collection.json

```diff
@@ -4235,7 +4235,7 @@
 			"name": "Liabilities",
 			"item": [
 				{
-					"name": "OK",
+					"name": "Retrieve Liabilities",
```

Comment:
I noticed sometimes we use "OK" and sometimes we use the endpoint name. Are we gradually renaming these across the json file?

Reply:
They are supposed to read "OK". I'll fix that.